.github/workflows/build-opencv-base-container.yml

```diff
@@ -10,6 +10,7 @@ on:
     - build/intermediate-containers.mk
     - Makefile
   pull_request:
+  types: [assigned, opened, synchronize, reopened, labeled, unlabeled]

```

Comment:
is `types` a child of `pull_requests`?

Reply:
(the indentation looks off to me)

Reply:
yes, types here is the activity type of the pull request. If there is a pull request event with these activity types, it would trigger the task. By default, workflow registering to pull request event will get callback to these 4 types (assigned, opened, synchronize, reopened). I added labeled and unlabeled so that when a label is added to or removed from a PR, we will get the callback.